### DISCUSSION TITLE
How to remove an account from the CLI?

### DISCUSSION THREAD

**Discussion by airblade** [ente-io/ente#1702] [Q&A]
I added a couple of ente accounts to the CLI via `ente account add ...`.

However now I only want to export from one of the accounts.  How can I remove the other from the accounts that the CLI knows about?

There is no documented `ente account remove ...` command so for now I guess the easiest way is to remove the CLI's configuration, to reset it, and then add the account I want to export from.  But I don't know where to find the configuration.

Any help would be much appreciated!

**Comment by ua741** [comment]
Hello, 
We will keep this issue open until we add a command to remove an account.
If you wish, you can remove all accounts and related details by delete `.ente` folder inside your home directory.

**Reply by airblade** [reply]
Thank you!

**Reply by jdhalbert** [reply]
Noting for future reference by folks reading this: removing an account by deleting the `.db` file does result in a re-download of the full photo library if the same account is reconfigured later, even if previously exported files are still in the destination. (Similarities to https://github.com/ente-io/ente/discussions/8453.)
